src/components/Editor/Tab.js

```diff
@@ -192,6 +192,7 @@ class Tab extends PureComponent<Props> {
         onContextMenu={e => this.onTabContextMenu(e, sourceId)}
         title={getFileURL(source)}
       >
+        <span className="devtools-tab-line"></span>
```

Comment:
Using a decorative span is alright, but can we use a class such as `source-tab-line` since it's a visual element of the `source-tab` component?